recipe/meta.yaml

```diff
@@ -1,17 +1,26 @@
-{% set version = "3.0.0b3" %}
-{% set version_str = "v3.0.0-beta-3" %}
+{% set name = "protobuf" %}
+{% set version = "3.0.0" %}
+{% set version_str = "v3.0.0" %}
+{% set sha256 = "f5b3563f118f1d3d6e001705fa7082e8fc3bda50038ac3dff787650795734146" %}
```

Comment:
People started doing it a long time ago and it has taken hold. Here's one of the first [conversations]( https://github.com/conda-forge/staged-recipes/pull/177#r57006776 ) on this point.

The argument has generally been it is easier to update with all this info at the top. We have even made this part of the example thanks to PR ( https://github.com/conda-forge/staged-recipes/pull/1052 ). This is just continuing the trend.